plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKudu115SmokeTests.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kudu;
+
+public class TestKudu115SmokeTests
```

Comment:
Instead of version numbers in name we can use KuduLatestSmokeTests and KuduSmokeTests (existing convention for other connectors) since we'll never test with more than 2 versions (most likely).